### ISSUE TITLE
Management-bar - links and buttons

### ISSUE THREAD

**Issue by marcoscv-work** [liferay/clay#1146]
Some actions should be buttons instead of links:

![image](https://user-images.githubusercontent.com/7963804/44907940-10f44900-ad1a-11e8-8af9-012e016a1750.png)

![image](https://user-images.githubusercontent.com/7963804/44907990-33866200-ad1a-11e8-993a-47f2820411d0.png)

**Comment by carloslancha** [comment]
There're cases where this should be links and other where should be buttons. For demo (and a11y tests) purposes just change the href for a real one.
